docs/user/alerting/alerting-troubleshooting.asciidoc

```diff
@@ -9,6 +9,22 @@ If your problem isn’t described here, please review open issues in the followi
 
 Have a question? Contact us in the https://discuss.elastic.co/[discuss forum].
 
+[float]
+[[rule-fails-to-decrypt-api-key]]
+=== Failing to decrypt apiKey
+
+*Problem*:
+
+The rule fails to execute and has an `Unable to decrypt attribute "apiKey"` error.
+
+*Solution*:
+
+This error happens when the `xpack.encryptedSavedObjects.encryptionKey` value changed since creating the rule. Depending on the scenario, there are different ways to solve this problem:
+
+* Suppose the value in `xpack.encryptedSavedObjects.encryptionKey` was manually changed, and the previous encryption key is still known. In that case, any previous encryption key needs to be part of the <decryption only key, encryption-key-rotation>.
+* Suppose another {kib} instance with a different encryption key connects to the cluster. In that case, it probably created a rule with a different encryption key. Ensuring the encryption keys between all the {kib} instances are the same is necessary, and setting <decryption only keys, encryption-key-rotation> for previously used encryption keys.
```

Comment:
```suggestion
* Suppose another {kib} instance with a different encryption key connects to the cluster. In that case, it may be trying to run the rule using a different encryption key than what the rule was created with. Ensuring the encryption keys between all the {kib} instances are the same is necessary, and setting <decryption only keys, encryption-key-rotation> for previously used encryption keys.
```

Reply:
Maybe you don't need the second sentence "In that case, .."  because it is explained as part of the solution.

Reply:
As discussed with @gchaps offline, we kept the sentence but changed the wording slightly.